### DISCUSSION TITLE
Re-configuration URL: Features and functionalities

### DISCUSSION THREAD

**Discussion by Rohanlochen** [SafeExamBrowser/seb-win-refactoring#583] [Q&A] [answered]
Hello,

I'm writing to ask some questions regarding the reconfiguration url and how it works, we understand the preliminary function of it; that it re-configures the .SEB file, however, how are these files distributed?

We are using Moodle 4.0 and in our proof of concept testing we upload the .SEB file to the "require use of safe exam browser" tab - does the reconfiguration URL change the settings of the current exam in a live event after the students have attempted a quiz or will a student's current attempt need to be deleted and an updated version of that configuration be in the new attempt or does the reconfiguration URL allow for 'on-the-fly' reconfiguration whilst an active attempt is in progress.

Thank you for your help.

Kindest regards,
Rohan

**Comment by danschlet** [comment, marked as answer]
We don't have time right now to explain this in detail. I recommend to read the manuals thoroughly and try to find it out by trial-and-error. You can load configuration files using sebs:// links (also by using a forward in the web server) or by serving them to be downloaded. SEB will then recognize that it's loading a .seb file and if reconfiguring is allowed, use the settings from the new configuration file. If the user is logged in and should stay logged in after the reconfiguration, you need to make sure to set the cookie settings correctly. See settings in [Exam tab](https://safeexambrowser.org/windows/win_usermanual_en.html#ExamPane).
